.github/workflows/ci.yml

```diff
@@ -5,6 +5,8 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    env:
+      LC_ALL: C
```

Comment:
Would it make sense to add this to Makefile.am? This way CI and manual invocation `make check-local` can be covered? AFAIK the check-news target is the only one, that requires LC_ALL=C. There are other parts in Makefile.am, where LC_ALL is set accordingly (search for LC_ALL).

Reply:
looking deeper, it is the `check-news` target, that enforces the C locale: https://github.com/ganeti/ganeti/blob/72fcdea9139c22201d6b19cc60a0525c2a345fe4/autotools/check-news#L97-L101
Because `check-local` should run by CI and locally (includes `check-news`) I suggest to fix this in `Makefile.am`:
```make
check-news:
        LC_ALL=C RELEASE=$(PACKAGE_VERSION) $(CHECK_NEWS) < $(top_srcdir)/NEWS
```

Reply:
I was wondering: could there be any other parts of the build process which actually rely on LC_ALL=C? Just setting it for a specific build step might break things at a later point.

Reply:
I think it would be a good idea to make entire build work work an UTF-8 locale, even if C.UTF-8. Plain C seems so archaic :)

Reply:
Sounds reasonable to me. But I would propose a separate PR for this. I suppose this should receive some through testing (especially regarding documentation). In any case I'd take a shot at it.

Reply:
Oh, definitely didn't mean in this PR. Was just surprised seeing we rely on C...